src/Amazon.Extensions.CognitoAuthentication/CognitoUser.cs

```diff
@@ -361,7 +499,20 @@ public virtual async Task SetUserSettingsAsync(IDictionary<string, string> userS
         /// <param name="paginationToken">Token to continue earlier search</param>
         /// <returns>Returns a list of CognitoDevices associated with this user</returns>
         [Obsolete("This method is deprecated since it only lists the first page of devices. The method ListDevicesV2Async should be used instead which allows listing additional pages of devices.")]
-        public virtual async Task<List<CognitoDevice>> ListDevicesAsync(int limit, string paginationToken, CancellationToken cancellationToken = default)
+        public virtual async Task<List<CognitoDevice>> ListDevicesAsync(int limit, string paginationToken)
+        {
+            return await ListDevicesAsync(limit, paginationToken);

```

Comment:
Oops, this is an infinite loop.

Reply:
Fixed in https://github.com/aws/aws-sdk-net-extensions-cognito/compare/1f63d79f69d9c0c8637f995deea0645d60fd4e01..4c21b8891e02a6322ae1c18bc1d1d6a14ce1854b